Competition2017/src/competition/subsystems/drive/commands/DriveForDistanceCommand.java

```diff
@@ -12,12 +12,14 @@
 import xbot.common.properties.DoubleProperty;
 import xbot.common.properties.XPropertyManager;
 import competition.subsystems.drive.DriveSubsystem;
+import competition.subsystems.pose.PoseSubsystem;
 
 public class DriveForDistanceCommand extends BaseCommand {
     private static Logger log = Logger.getLogger(DriveForDistanceCommand.class);
     
     private final DriveSubsystem driveSubsystem;
     private final PIDManager travelManager;
+    private final PoseSubsystem pose;
```

Comment:
It'd be easier to track these variables if subsystems always ended in `subsystem`.